pandas/tests/arrays/test_array.py

```diff
@@ -0,0 +1,158 @@
+import decimal
+
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.dtypes import registry
+
+import pandas as pd
+from pandas.api.extensions import register_extension_dtype
+from pandas.core.arrays import integer_array, period_array
+from pandas.tests.extension.decimal import (
+    DecimalArray, DecimalDtype, to_decimal)
+import pandas.util.testing as tm
+
+
+@pytest.mark.parametrize("data, dtype, expected", [
+    # Basic NumPy defaults.
+    ([1, 2], None, np.array([1, 2])),
+    ([1, 2], object, np.array([1, 2], dtype=object)),
+    ([1, 2], np.dtype('float32'),
+     np.array([1., 2.0], dtype=np.dtype('float32'))),
+    (np.array([1, 2]), None, np.array([1, 2])),
+    # String alias passes through to NumPy
+    ([1, 2], 'float32', np.array([1, 2], dtype='float32')),
+    # Period alias
```

Comment:
I would just put a blank line between cases, I think that helps a lot (e.g. before the comment)